CMakeLists.txt

```diff
@@ -267,36 +254,31 @@ if (APPLE OR ANDROID)
       OUTPUT_STRIP_TRAILING_WHITESPACE)
   endif()
   if (WITH_STATIC)
-    #set(CMAKE_FIND_LIBRARY_SUFFIXES .a ${CMAKE_FIND_LIBRARY_SUFFIXES})
-    set(CMAKE_FIND_LIBRARY_SUFFIXES .a)
+    set(OPENSSL_USE_STATIC_LIBS TRUE)
```

Comment:
OpenSSL is no longer linked statically when building `release-static`. I've only tested on linux and mingw but I imagine this applies to all builds. This line may be the cause or possibly when you decoupled cpp-netlib from `add_subdirectory()`. TBD, but I'll let you find out why.

Reply:
Because cpp-netlib is built via `add_submodule()`, for static linking we need to specify the cmake option `WITH_STATIC_DEPS=ON`. I will fix Makefile accordingly.